RazorDiagnostics/CollectCurrentFileDiagnostics.cs

```diff
@@ -91,12 +90,45 @@ private void Execute(object sender, EventArgs e)
         {
             ThreadHelper.ThrowIfNotOnUIThread();
 
-            OutputWindow.Show();
-            OutputWindow.OutputString($"Visual Studio Binary in {InstallationStateUtilities.IDEPath}:\r\n    {InstallationStateUtilities.IDEInfo}\r\n");
-            OutputWindow.OutputString($"Is Razor Dependency Installed:\r\n    {InstallationStateUtilities.IsRazorDependencyInstalled}\r\n");
-            OutputWindow.OutputString($"Installed Web Editor Assemblies in {InstallationStateUtilities.WebEditorFolder}: \r\n{InstallationStateUtilities.InstalledWebEditorAssemblies}\r\n");
-            OutputWindow.OutputString($"Installed Web Editor .Net Core Razor Assemblies in {InstallationStateUtilities.WebEditorRazor4Folder}: \r\n{InstallationStateUtilities.InstalledWebEditorRazorV4Assemblies}\r\n");
-            OutputWindow.OutputString($"Installed Razor Extension Assemblies in {InstallationStateUtilities.RazorExtensionFolder}: \r\n{InstallationStateUtilities.InstalledRazorExtensionAssemblies}\r\n");
+            try
+            {
+                OutputWindow.Show();
+                OutputWindow.OutputString($"Visual Studio Binary in {InstallationStateUtilities.IDEPath}:\r\n    {InstallationStateUtilities.IDEInfo}\r\n");
+                OutputWindow.OutputString($"Is Razor Dependency Installed:\r\n    {InstallationStateUtilities.IsRazorDependencyInstalled}\r\n");
+                OutputWindow.OutputString($"Installed Web Editor Assemblies in {InstallationStateUtilities.WebEditorFolder}: \r\n{InstallationStateUtilities.InstalledWebEditorAssemblies}\r\n");
+                OutputWindow.OutputString($"Installed Web Editor .Net Core Razor Assemblies in {InstallationStateUtilities.WebEditorRazor4Folder}: \r\n{InstallationStateUtilities.InstalledWebEditorRazorV4Assemblies}\r\n");
+                OutputWindow.OutputString($"Installed Razor Extension Assemblies in {InstallationStateUtilities.RazorExtensionFolder}: \r\n{InstallationStateUtilities.InstalledRazorExtensionAssemblies}\r\n");
+
+                PopSuccessMessage();
+                OutputWindow.Show();
+            }
+            catch (Exception ex)
+            {
+                System.Diagnostics.Debug.WriteLine("Exception occurred while attempting to output Razor diagnostics.", ex);
+                PopFailureMessage(ex.Message);
+            }
         }
+
+        private static void PopSuccessMessage()
```

Comment:
Presumably the user will have to do this only once, otherwise the message might start getting annoying. Lets leave it in and see if we get user feedback on that.